static/js/hotkey.js

```diff
@@ -591,7 +586,7 @@ exports.process_hotkey = function (e, hotkey) {
             return true;
     }
 
-    if (current_msg_list.empty()) {
+    if (current_msg_list.empty() && event_name !== 'reply_message') {
```

Comment:
I think we should just handle `reply_message` higher up in the function now, so that we don't need to make this exception for it.